### DISCUSSION TITLE
Authenticator in steamctl and steam app at the same time without root?

### DISCUSSION THREAD

**Discussion by Gus2002** [ValvePython/steamctl#61] [Q&A] [answered]
I noticed the preview section of the readme is showcasing authenticator code being generated both in steamctl and steam mobile app without root access. Is there a tutorial on how to get this setup?

**Comment by rossengeorgiev** [comment, marked as answer]
When you start the add authenticator process, you will be guided via prompts in order to configure both if you choose.

**Reply by rossengeorgiev** [reply]
This comment described the steps: https://github.com/ValvePython/steamctl/discussions/34#discussioncomment-627095
